src/combobox/example/index.ts

```diff
@@ -142,7 +142,9 @@ export class App extends WidgetBase<WidgetProperties> {
 	}
 
 	render(): DNode {
-		return v('div', [
+		return v('div', {
+			styles: { maxWidth: '256px' }
```

Comment:
Based on talking with Scott, I believe we decided no max width. It makes a certain sort of sense -- desired width can be really variable (no pun intended, mostly), and controlling input width through styles on the containing element seems like a better pattern anyway.